src/index.ts

```diff
@@ -11,61 +11,18 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// ** This file is automatically generated by synthtool. **
```

Comment:
@bcoe Here's another instance of index.ts differing in a bad way.

Is `hermetic_postprocess_gapic_library()` doing something different than `postprocess_gapic_library()`?

Reply:
The problem here is that the `index.ts` is handwritten, and was excluded in `synth.py`. When no default_version or versions are passed to the Node templates, they omit generating `index.ts`:

https://github.com/googleapis/nodejs-translate/blob/master/synth.py#L35